scripts/f5-config.py

```diff
@@ -14,31 +14,57 @@
 # limitations under the License.
 #
 # (c) 2014, Kevin Carter <kevin.carter@rackspace.com>
+from __future__ import print_function
 import argparse
 import json
 import os
+import sys
 
 import netaddr
 
+IS_LEM = True
 
-PART = 'RPC'
-PREFIX_NAME = 'RPC'
+if 'LAB_DATA_DIR' in os.environ:
+    # If we have LAD_DATA_DIR set, we are operating inside a lab environment
+    LAB_DATA_DIR = os.environ['LAB_DATA_DIR']
```

Comment:
I think the envar is not defined you get a keyerror `KeyError: 'LAB_DATA_DIR'`

the second part seems good though